idaes/core/util/model_statistics.py

```diff
@@ -377,7 +378,31 @@ def number_activated_equalities(block):
     Returns:
         Number of activated equality Constraint components in block
     """
-    return sum(1 for _ in activated_equalities_generator(block))
+    return sum(
+        1 for _ in activated_equalities_generator(block)
+    ) + number_grey_box_equalities(block)
+
+
+def number_grey_box_equalities(block):
```

Comment:
`number_of_X` is less ambiguous in English.
Can add `-> int` to indicate return type and, if it's possible, the expected type of `block`.
In docs: "Method to" is superfluous and at any rate this is a function.

Reply:
done